### DISCUSSION TITLE
[BUG] Connect-PnPOnline -Interactive (Error: Navigation to the webpage was canceled)

### DISCUSSION THREAD

**Discussion by KeyCode-brenden** [pnp/powershell#4212] [General]
### Reporting an Issue or Missing Feature
Please confirm what it is that your reporting
> When calling "Connect-PnPOnline -Url $siteURL -Interactive" the interactive window that appears displays "Navigation to the webpage was canceled" 

### Expected behavior 
> Interactive window would open and I would be able to login with my password and 2FA

### Actual behavior 
> Interactive window displays the error "Navigation to the webpage was canceled" 
> This script was working without issue last week
![image](https://github.com/user-attachments/assets/e57321d7-d580-4600-adc1-ac9860727b88)
> A message in the console will also be displayed mentioning the need for an EntraID App registration, this error happens if I attempt the registration process as well. 
> Reproduceable in Windows Sandbox 

### Steps to reproduce behavior
$siteURL = "https://exampletenant.sharepoint.com/"
$connection = Connect-PnPOnline -Url $siteURL -Interactive -ForceAuthentication -ReturnConnection

### What is the version of the Cmdlet module you are running?
2.9.0
PowerShell 7

### Which operating system/environment are you running PnP PowerShell on?
- [X] Windows

**Comment by gautamdsheth** [comment]
Can’t reproduce it, have been using it quite extensively for the past few days.

can you please update to latest nightly builds and try ?

**Comment by GitElk** [comment]
Same issue using 
Connect-PnPOnline -Url $URL -Tenant $CorpTenantID -Interactive -ClientId $CollabInteractive 
PSVersion                      7.4.6       PnP 2.12
this worked a week ago. (clientID needed with recent security changes for interactive logon)
It does, however run from my laptop rather than server on same pnp / ps versions) and asks which account to use.

**Comment by CrazyDindon** [comment]
Have the issue on 2 fev 2025, was working on 20-12-2024.
Connect-PnPOnline -url $SiteUrl -ClientId $ClientID-Interactive
Authent Popup show only late with this error.
PowerShell 7.4.6
PnP.PowerShell 2.12.0

**Reply by CrazyDindon** [reply]
Fixed it on my side. It seems that the authent page was blocked locally, not sure why.
I added urls to internet options trusted sites (but also restarted with a windows update)
Urls added:

- https://login.microsoftonline.com/
- https://aadcdn.msftauth.net/
- localhost
